### PR TITLE
Support to bridges with external sets/functions

### PR DESCRIPTION
**Pull request by IssamT**
It is working for what I needed but I'd consider the PR as a WIP. There are still two issues mainly:

- The sets/functions need to be explicitly imported in the current scope.
- I had no clue how to name the new equivalent functions.

### CODE REVIEW

File: src/Utilities/model.jl [hunk 1]
```diff
@@ -410,14 +410,31 @@ _mod(m::Module, s::Symbol) = Expr(:., m, :($(QuoteNode(s))))
 # (:Zeros) -> :(MOI.Zeros)
 _moi(s::Symbol) = _mod(MOI, s)
```
Comment (blegat) [inline src/Utilities/model.jl]: Why not simply removing this function and force the user to write
```julia
@model LPModel () (MOI.EqualTo, MOI.GreaterThan, MOI.LessThan, MOI.Interval) (MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives) () (MOI.SingleVariable,) (MOI.ScalarAffineFunction,) (MOI.VectorOfVariables,) (MOI.VectorAffineFunction,)
```

Comment (blegat) [reply src/Utilities/model.jl]: If you remove this _moi function and all calls to it then it should work

Comment (mlubin) [reply src/Utilities/model.jl]: FWIW the escaping/scoping rules for macros changed between 0.6 and 0.7. You might run into this.

Comment (odow) [reply src/Utilities/model.jl]: Forcing the module prefix sounds like the most robust way to go.

**Comment by IssamT** [comment]
replaced by #510

**Pull request closed without merging**